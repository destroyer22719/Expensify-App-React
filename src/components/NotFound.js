import React from 'react';

const NotFound = () => (
  <div>This is Not Found <br /> Go to <Link to="/">Go Home</Link> Page</div>
)

export default NotFound