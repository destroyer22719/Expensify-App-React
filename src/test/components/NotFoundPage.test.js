import React from 'react';
import { shallow } from 'enzyme';
import NotFound from '../../components/NotFound';
import "../setupTest";

test('testing the not found page to work properly', () => {
  const wrapper = shallow(<NotFound />);
  expect(wrapper).toMatchSnapshot();
});
