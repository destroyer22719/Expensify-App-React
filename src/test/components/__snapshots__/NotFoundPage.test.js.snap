// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`testing the not found page 1`] = `
<div>
  This is Not Found 
  <br />
   Go to 
  <Link
    to="/"
  >
    Go Home
  </Link>
   Page
</div>
`;

exports[`testing the not found page to work properly 1`] = `
<div>
  This is Not Found 
  <br />
   Go to 
  <Link
    to="/"
  >
    Go Home
  </Link>
   Page
</div>
`;
